CHANGELOG.md

```diff
@@ -2,31 +2,27 @@
 
 ## Upcoming Release
 
-This upcoming release of the agent adds instrumentation for Active Support caching operations, Action Mailer operations, and Action Mailbox operations.
+The upcoming release of the agent introduces additional Ruby on Rails instrumentation (especially for Rails 6 and 7) for various Action*/Active* libraries whose actions produce Active Support notifications events.
 
-- **Add Active Support Instrumentation**
+- **Add Various Additional Ruby on Rails Library Instrumentations**
 
-  Instrumentation is now automatically provided for all [Active Support caching](https://guides.rubyonrails.org/caching_with_rails.html) operations. Whenever a caching operation is performed, a New Relic segment is created that contains timing information as well as parameters for the cache key, store, and other relevant data. [PR#1742](https://github.com/newrelic/newrelic-ruby-agent/pull/1742)
+  New instrumentation is now automatically provided by several Action*/Active* libaries that generate Active Support notifications. With each new Ruby on Rails release, new events are added to the Rails libraries and sometimes existing events have their payload parameters updated as well. The New Relic Ruby agent will now automatically process more of these events and parameters with New Relic segments that are created for each event. At a minimum, each segment will provide timing information for the event. In several cases, all non-sensitive event payload parameters are also passed along in the segment.
 
-  | Configuration name | Default | Behavior |
-  | ----- | ----- | ----- |
-  | `disable_active_support` | `false` | If `true`, disables Active Support instrumentation. |
+  The agent now newly supports or has updated support for the following libraries:
 
-- **Add Action Mailer Instrumentation**
+  - Action Dispatch (for middleware)[PR#1745] (https://github.com/newrelic/newrelic-ruby-agent/pull/1745)
```

Comment:
There's an extra space after [PR#1745] that's preventing the link from connecting

Reply:
Thanks for resolving this with [5f6ca32](https://github.com/newrelic/newrelic-ruby-agent/pull/1745/commits/5f6ca32a42d4bf81c900fff962d450e8624bfc0c)